packages/core/forms/src/components/fields/FieldInput.vue

```diff
@@ -85,6 +102,14 @@ const emit = defineEmits<{
 }>()
 
 const propsRefs = toRefs(props)
+const eyeOpen = ref(false)
```

Comment:
nit: Would you mind changing `eyeOpen` to `maskPassword` for a clearer meaning?

Reply:
Good point, merge it too soon. Since [Justineo](https://github.com/Justineo) will propose this as built-in Kongponents, i think we'll be refactoring this in near future.